packages/grid/src/GridMetricCalculator.ts

```diff
@@ -1043,7 +1043,7 @@ export class GridMetricCalculator {
         firstColumn,
         treePaddingX
       );
-      columnWidths.set(column, columnWidth);
+      columnWidths.set(column, Math.min(600, columnWidth));
```

Comment:
We want this constant to be defined in the theme so it can be changed manually if needed. Take a look at the `calculateColumnWidth` method - we already have a `minColumnWidth`, we could just define a `maxColumnWidth` that it uses and never go over that.
Note this doesn't address https://github.com/deephaven/web-client-ui/issues/1284, but that's fine and can be addressed in a separate PR.

Reply:
I can take on #1284 after this one. I think it'll be somewhat related anyways